File: src/main/java/com/hy/frame/audio/AudioParam.java
package com.hy.frame.audio;

public class AudioParam {

    private int mFrequency; // 采样率

    private int mChannel; // 声道

    private int mSampBit; // 采样精度

    /**
     * 采样率
     */
    public int getmFrequency() {
        return mFrequency;
    }

    /**
     * 采样率
     */
    public void setmFrequency(int mFrequency) {
        this.mFrequency = mFrequency;
    }

    /**
     * 声道
     */
    public int getmChannel() {
        return mChannel;
    }

    /**
     * 声道
     */
    public void setmChannel(int mChannel) {
        this.mChannel = mChannel;
    }

    /**
     * 采样精度
     */
    public int getmSampBit() {
        return mSampBit;
    }

    /**
     * 采样精度
     */
    public void setmSampBit(int mSampBit) {
        this.mSampBit = mSampBit;
    }

}

File: src/main/java/com/hy/frame/audio/AudioParam.kt
package com.hy.frame.audio

class AudioParam {

    private var mFrequency: Int = 0 // 采样率

    private var mChannel: Int = 0 // 声道

    private var mSampBit: Int = 0 // 采样精度

    /**
     * 采样率
     */
    fun getmFrequency(): Int {
        return mFrequency
    }

    /**
     * 采样率
     */
    fun setmFrequency(mFrequency: Int) {
        this.mFrequency = mFrequency
    }

    /**
     * 声道
     */
    fun getmChannel(): Int {
        return mChannel
    }

    /**
     * 声道
     */
    fun setmChannel(mChannel: Int) {
        this.mChannel = mChannel
    }

    /**
     * 采样精度
     */
    fun getmSampBit(): Int {
        return mSampBit
    }

    /**
     * 采样精度
     */
    fun setmSampBit(mSampBit: Int) {
        this.mSampBit = mSampBit
    }

}